spinn_front_end_common/interface/spinnaker.cfg

```diff
@@ -106,6 +106,8 @@ spalloc_machine = None
 
 virtual_board = False
 requires_wrap_arounds = None
+NCoresPerChip = 18
+RouterTableEntriesPerRouter = 1024
```

Comment:
Should this be 1023 to work with "normal" circumstances i.e. one entry taken by SCAMP?

Reply:
well i changed it, but the default router says there's 1024 entries. as line 17 of router states. So we're always routing with 1024 currently. but changed anyhow.